packages/eslint-plugin/tests/rules/switch-exhaustiveness-check.test.ts

```diff
@@ -142,7 +142,7 @@ function test(value: Union): number {
   }
 }
     `,
-    // Switch contains default clause.
+    // Switch contains default clause
```

Comment:
[Proofreading] Unrelated changes. If you think we should standardize when/whether comments have punctutation / start with capital or lowercase letters then that'd be a separate issue to discuss from this PR. For now it'd be better (faster to review, cleaner diffs) to not make changes to code that wasn't otherwise touched.

Reply:
i looked through the comments and most of them did not have punctuation, so i standardized the style, which was only an extra 4 lines changed.
if you want i can remove those 4 changes from the PR?

Reply:
Yes please :)